crawl-ref/source/dgn-overview.cc

```diff
@@ -25,11 +26,27 @@
 #include "religion.h"
 #include "scroller.h"
 #include "stairs.h"
+#include "store.h" //for level_id()
 #include "stringutil.h"
 #include "terrain.h"
 #include "travel.h"
 #include "unicode.h"
 
+enum AnnotationPossibilities
+{
+    // Annotate the dungeon floor the player character is currently on
+    ID_HERE     = -100,
+
+    // Annotate one level up
+    ID_UP       = -99,
```

Comment:
The sequencing here is odd, it looks like maybe the intent is to be able to meaningfully compare ID_HERE, UP, and DOWN using int comparison, but I would basically recommend against ever trying to build that directly into enum values. (For one thing, if they ever become an enum class it's impossible.) Of course if you reorder these to put ID_UP first I wouldn't even have commented on this :-)